client/src/utils/api.js

```diff
@@ -0,0 +1,11 @@
+export async function getGroceriesHeader() {
+  const result = await fetch(`http://localhost:3001/api/groceries`);
+  const groceriesHeader = result.json();
+  return groceriesHeader;
+}
+
+export async function getGroceries() {
+  const result = await fetch(`http://localhost:3001/api/groceries`);
+  const groceries = result.json();
+  return groceries;
+}
```

Comment:
you have to wait for the response to come back, before transforming it to and js object
```suggestion
export async function getGroceries() {
  const response = await fetch(`http://localhost:3001/api/groceries`);
  const data = await response.json();
  return data;
}
```

Reply:
Check out the project from our OMR's:
https://github.com/miriamhaenle/omr-fetch-demo/blob/main/src/api/fetchData.js

1. Awaiting the `response.json()` is not necessary. 
2. However it doesn't help with the issue to render the content of my db.json onto my page.

Reply:
sorry, just common practice 🤷‍♂️